crawler_mcp/crawlers/optimized/run.py

```
@@ -0,0 +1,1216 @@
+    )
+
+    strat = OptimizedCrawlerStrategy(cfg)
+
+    # Optional: human-friendly per-page logs via monitoring hooks
+    if args.per_page_log or show_progress:
+
+        def _print(s: str) -> None:
+            try:
+                print(s, flush=True)
+            except Exception:
+                pass
```

Ruff (0.12.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)